recipes/prolif/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "prolif" %}
+{% set version = "0.3.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Pypi release is outdated. See https://github.com/chemosim-lab/ProLIF/issues/6
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/prolif-{{ version }}.tar.gz
+  # sha256: 82b246a300c33306b1da8768d39e9edc2eff6dfbc37d7c764856d8bc2bade0f9
+
+  url: https://github.com/chemosim-lab/ProLIF/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1d8fb97ea55e727e64f118ea05cfac2a6eec7ef1887f89a5df27a5be7a23d49a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - rdkit
+  run:
+    - python >=3.6
```

Comment:
```suggestion
    - python >=3.8
```

Reply:
The runner will set the dep required to be able to install prolif. So it's up to you to decide what min python version prolif is compatible with. If it's compatible with 3.6/3.7, then I would simply set 3.6/3.7.

Reply:
Oh sorry I thought you were talking about the `run` section when you said "the runner".

Reply:
Yeah sorry I meant `host` :') thanks!